src/components/custom/_dhero-2.js

```diff
@@ -202,9 +174,54 @@ const query = graphql`
         deriv_x_mobile: file(relativePath: { eq: "deriv-x/hero-laptop-mobile.png" }) {
             ...bannerImage
         }
+        qr_code: file(relativePath: { eq: "deriv-go/deriv_go_all_appstores.png" }) {
+            ...fadeIn
+        }
+        web_browser: file(relativePath: { eq: "deriv-go/web-browser.png" }) {
+            ...fadeIn
+        }
     }
 `
+const AppButton = styled(LocalizedLink)`
+    margin-right: 8px;
+    padding: 0;
+    border: none;
 
+    img {
+        border-radius: 7px;
+    }
+    @media ${device.tabletL} {
+        margin-bottom: 8px;
+        width: 156px;
+        height: 46px;
+    }
+    @media ${device.mobileL} {
+        width: 150px;
+        height: 40px;
+    }
+`
+const ButtonDp2p = styled(Button)`
+    opacity: 0;
+    @media ${device.tablet} {
+        opacity: 1;
+        padding: 1.5rem 1.6rem;
+        height: 42px;
+        width: 25rem;
+        white-space: nowrap;
+        margin-right: 50%;
+        margin-top: 24px;
+        margin-bottom: 40px;
+    }
+    @media (min-width: 768px) and (max-width: 1023px) {
+        opacity: 1;
+        padding: 1.5rem 1.6rem;
+        height: 42px;
+        white-space: nowrap;
+        margin-right: 500px;
```

Comment:
why this large margin right? Can't this be managed by flex?

Reply:
That too turns out that it's not needed. Could be done with flex better idea. Thanks